src/web/routes/figma/integration.test.ts

```diff
@@ -219,6 +225,19 @@ describe('/figma', () => {
 				);
 			});
 
+			it('should return a 200 if the passcode is invalid', async () => {
```

Comment:
**suggestion:** Per Figma's security recommendation, we need to return `HTTP 400` in this case:

> We would recommend comparing the passcode we pass back to you with the passcode originally provided when creating the endpoint to make sure they match before acting on the webhook trigger. If you receive a request with the wrong passcode, you should respond with a 400 Bad Request HTTP response which will immediately stop the webhook. ([source](https://www.figma.com/developers/api#webhooks-v2-security)).

Reply:
Interesting, I wonder if `stop the webhook` means it no longer fires events. If it does, I think I would prefer to return a `200` to avoid potential for abuse. Will do some testing on this.

Reply:
Hey Justin! This was my assumption too but I was not able to reproduce this behavior -- I keep receiving events after `HTTP 400`. We could double-check this with the Figma team.